tools/scylla-sstable.cc

```diff
@@ -2616,6 +2615,30 @@ void shard_of_operation(schema_ptr, reader_permit,
     writer.EndStream();
 }
 
+void shard_of_with_tablets() {
+    // TODO: support tablets
+    fmt::print(std::cerr, "tablets not supported");
+}
+
+void shard_of_operation(schema_ptr, reader_permit,
+                        const std::vector<sstables::shared_sstable>& sstables,
+                        sstables::sstables_manager& sstable_manager,
+                        const bpo::variables_map& vm) {
+    // uses "tablets" by default. as new scylla installations enable "tablet" by
+    // default
+    if (vm.count("tablets") && vm.count("vnodes")) {
+        throw std::invalid_argument("'--tablets' and '--vnodes' are mutual exclusive");
```

Comment:
s/mutual/mutually

Reply:
Also, I think we don't use `'` around option names, elsewhere in `scylla-sstable` (also below).

Reply:
ack. and dropped.